native/packages/react-native-backpack/libraries/Badge/ios/react/classes/BPKBadgeWrapper.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <React/RCTView.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@class RCTBridge;
+@interface BPKBadgeWrapper : RCTView
+- (instancetype)initWithBridge:(RCTBridge *)bridge NS_DESIGNATED_INITIALIZER;
```

Comment:
What is `NS_DESIGNATED_INITIALIZER`?

Reply:
See https://stackoverflow.com/questions/26185239/ios-designated-initializers-using-ns-designated-initializer